### PR TITLE
Ckwu feat experiment

### PR DESCRIPTION
**Pull request by Brian-Ckwu**

### CODE REVIEW

File: experiment.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        # TODO: save experiment config (yml)
+        # Cost estimation
+        cost = self.estimate_cost()
+        flag = input(f"Estimated cost: {cost} USD (~{cost * 30} TWD). Continue? (Y)")
+        if flag != 'Y':
+            logging.info("Experiment aborted.")
+            return
+        
+        for i, pat in enumerate(self._samples.itertuples()):
+            if (i < 63): # NOTE: conduct the subset
```
Comment (wlchen0206) [inline experiment.py:144]: What dose 63 means?

**Review by wlchen0206 — COMMENTED** [review]
I leave one comment~

File: experiment.py (hunk 1 above)
Comment (Brian-Ckwu) [reply experiment.py:144]: Haha it is just an API "breakpoint" in an experiment. I forgot to delete.

**Review by wlchen0206 — APPROVED** [review]
LGTM!!!

**Pull request merged**